src/test/java/core/reader/TabDelimitedReaderTest.java

```diff
@@ -0,0 +1,97 @@
+package core.reader;
+
+import static core.account.AccountSide.CREDIT;
+import static core.account.AccountSide.DEBIT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.Journal;
+import core.Ledger;
+import core.account.AccountDetails;
+import core.account.AccountDetailsImpl;
+import core.account.AccountingEntry;
+import core.chartofaccounts.ChartOfAccounts;
+import core.transaction.AccountingTransaction;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+
+class TabDelimitedReaderTest {
+
+  @Test
+  void testReadAccounts() {
+
```

Comment:
The empty lines at the beginning and end of methods are not consistent with the style of the existing code. Also, for consistency, please add "Arrange", "Act" and "Assert" comments to the tests, e. g.:

```java
@Test
public void testFreeze() {
    // Arrange
    AccountingEntry entry1 = new AccountingEntry(new BigDecimal(20), "0001", DEBIT);
    AccountingEntry entry2 = new AccountingEntry(new BigDecimal(20), "0002", CREDIT);
    AccountingTransaction transaction = new AccountingTransaction(Sets.newHashSet(entry1, entry2));
    // Act
    Executable act = () -> entry1.setTransaction(transaction);
    // Assert
    assertThrows(IllegalStateException.class, act);
}
```